src/Sarif/OptionallyEmittedData.cs

```diff
@@ -79,6 +79,9 @@ public enum OptionallyEmittedData : int
 
         // Enrich SARIF log with git blame information
         GitBlameInformation = 0x1000,
+        
+        // Enrich SARIF log with partial fingerprint
+        PartialFingerprints = 0x2000,
```

Comment:
>PartialFingerprints [](http://example.com/codeflow?start=8&length=19)

PartialFingerprints can contains multiple entries, the context region hash is just one of them.

The enum entry name should be more specified, e.g. "ContextRegionSnippetHashFingerprints"

Reply:
What do you think about `ContextRegionSnippetPartialFingerprints`? I would like to have `PartialFingerprints` in the name to find this feature faster.

Reply:
Sounds good!